.github/workflows/frontend_ci.yaml

```diff
@@ -85,6 +84,7 @@ jobs:
   build:
     name: Test build
     runs-on: ubuntu-latest
+    needs: [lint, prettier, deadcode]
```

Comment:
po co to w sumie jest? raczej chcemy mieć jak najszybciej feedback, a nie że muszę czekać na formatowanie żeby wiedzieć czy moja apka się builduje